Automattic/wp-calypso
# Primary Site: re-direct WordPress.com to /sites

Related to:
- https://github.com/Automattic/wp-calypso/issues/92364

## Proposed Changes

* We now redirect WordPress.com to `/sites` when logging in.
* We've deleted the option to either start at `/sites` or the primary site.

## Why are these changes being made?


* Customers are confused when they have multiple sites. Since the primary site is the first created site, once a customer becomes a paid customer they're getting redirected to the main site which might be a free site instead of the paid site.

## Testing Instructions



* Open the Calypso live link
* Go to WordPress.com 
* You should be redirected to `/sites`
* Go to wordpress.com/me/account 
* There should not be an option to set the `/sites` view.
* Check for regressions when creating a new site or other possible scenarios.

(I will add some images or video later on)

## Pre-merge Checklist
